### DISCUSSION TITLE
Adding SAS cables and connectors

### DISCUSSION THREAD

**Discussion by 2code4me** [netbox-community/netbox#12203] [Ideas]
Hi,
I'm an EXTREMELY newbe on netbox. 
At the moment I'm adding lot of storage controllers and shelf (like Dell Data Domain or Netapp) and, of course, these devices use SAS cables and connectors but I can't find any of this kind so I'm creating new devices uses connector type "other" and for cables I'm using more generic "DAC"

Would be possible to add some SAS connectors like "miniSAS 26 Pin SFF-8088" or "miniSAS HD SFF-8644" ??

Thanks!

**Comment by mkarel** [comment]
I would agree that adding SAS Connectors to the list of interface types would be a great addition.  @jeremystretch would this be something you would consider?

I think these 3 would be the most popular external interface types
SFF-8644 (Mini SAS HD)
SFF-8088 (Mini SAS)
SFF-8470 (Infiniband)
